website/scripts/events-handler.js

```diff
@@ -14,13 +14,14 @@ function parseKeyboardMappings(text)
             continue
         }
         var lineSplit = line.split(':');
-        if(lineSplit.length != 2)
-        {
-            publichAlert("Bad formatting: "+line, 'danger')
-            publichAlert("Format should look like: key:note_id  ", 'warning')
-            publichAlert("Example: a:60  ", 'warning')
-            return null;
-        }
+        console.log(lineSplit);
+        // if(lineSplit.length != 2)
+        // {
+        //     publichAlert("Bad formatting: "+line, 'danger')
+        //     publichAlert("Format should look like: key:note_id  ", 'warning')
+        //     publichAlert("Example: a:60  ", 'warning')
+        //     return null;
+        // }
```

Comment:
Is there any reason to remove alters?

Reply:
You must separate the number pad from the number.
I got the code value, but the length exceeded 2, so I temporarily annotated it